klog.go

```diff
@@ -1265,6 +1265,36 @@ func Infof(format string, args ...interface{}) {
 	logging.printf(infoLog, logging.logr, format, args...)
 }
 
+// InfoS structured logs to the INFO log.
```

Comment:
Could you document how arguments will be handled to match Infof? 
Maybe add example of format in docstring.

Reply:
done